cluster/saltbase/salt/kube-addons/initd

```diff
@@ -44,14 +45,38 @@ do_start()
 #
 do_stop()
 {
-    kill $(cat ${PIDFILE})
-    rm ${PIDFILE}
+    # start-stop-daemon is not used because we have to stop all children
+    # limitations:
+    #  - stop does not work if the pid file is missing
+    #  - stop does not work if the daemon process is missing (children will not
+    #    be killed)
+    # This is sufficient - remaining processes will end after a while.
+
+    local pid
+    pid=$(cat "${PIDFILE}" 2> /dev/null)
+    if [[ $? != 0 ]]; then
+        return 1
+    fi
```

Comment:
Is there a reason these lines aren't spelled:
```
if ! pid=$(cat "${PIDFILE}" 2> /dev/null); then
  return 1
fi
```
?

It's more idiomatic to test for return inline unless you need to pocket the return value.